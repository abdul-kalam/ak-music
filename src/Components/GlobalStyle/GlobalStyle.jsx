import * as React from 'react';
import  {GeneralStyles } from './general.styles';

const GlobalStyle = () => (
  <>
    <GeneralStyles />
  </>
);

export default GlobalStyle;
