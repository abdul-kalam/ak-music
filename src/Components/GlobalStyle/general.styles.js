import { createGlobalStyle } from 'styled-components';
import reset from 'styled-reset';

export const GeneralStyles = createGlobalStyle`
${reset}
:root {
  --background-color-primary: #1f1f1f;
  --background-color-light: rgba(45, 45, 45);
  --redux-color: #764abc;
}
*{
    box-sizing: border-box;
  }
  html{
    scroll-behavior: smooth;
  }
  body {
    font-size: 16px;
    background-color: var(--background-color-primary);
    color: rgba(255, 255, 255, 0.85);
  }
  .ellipses {
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  }
  sup{
    vertical-align: super;
  }
  sub{
    vertical-align: sub;
  }
  em{
    font-style: italic;
  }
  pre{
    white-space: pre-wrap;
  }
  li {
    list-style: none;
  }
  *:focus {outline:none;}

`;

