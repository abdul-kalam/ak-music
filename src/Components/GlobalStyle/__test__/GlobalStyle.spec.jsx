import React from 'react';
import GlobalStyle from '..';
import renderer from 'react-test-renderer';

describe('render globalstyle component', () => {
  test('check globalstyle snapshot', () => {
    const tree = renderer.create(<GlobalStyle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
