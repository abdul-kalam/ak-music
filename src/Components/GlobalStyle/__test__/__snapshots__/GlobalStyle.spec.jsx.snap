// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render globalstyle component check globalstyle snapshot 1`] = `null`;
